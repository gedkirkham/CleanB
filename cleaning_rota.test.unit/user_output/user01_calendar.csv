Date,Kitchen,Bedroom,Lounge,Garage,
25/08/2018,Ged,Razvan,,,
01/09/2018,Razvan,,Ged,Razvan,
08/09/2018,Ged,Ged,,,
15/09/2018,Razvan,,,,
22/09/2018,Ged,Razvan,,,
29/09/2018,Razvan,,Razvan,Ged,
06/10/2018,Ged,Ged,,,
13/10/2018,Razvan,,,,
20/10/2018,Ged,Razvan,,,
27/10/2018,Razvan,,Ged,Razvan,
03/11/2018,Ged,Ged,,,
10/11/2018,Razvan,,,,
17/11/2018,Ged,Razvan,,,
24/11/2018,Razvan,,Razvan,Ged,
01/12/2018,Ged,Ged,,,
08/12/2018,Razvan,,,,
15/12/2018,Ged,Razvan,,,
22/12/2018,Razvan,,Ged,Razvan,
