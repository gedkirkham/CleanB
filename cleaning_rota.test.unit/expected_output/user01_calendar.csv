Date,Kitchen,Bedroom,Lounge,Garage,
18/08/2018,Ged,Razvan,,,
25/08/2018,Razvan,,Ged,Razvan,
01/09/2018,Ged,Ged,,,
08/09/2018,Razvan,,,,
15/09/2018,Ged,Razvan,,,
22/09/2018,Razvan,,Razvan,Ged,
29/09/2018,Ged,Ged,,,
06/10/2018,Razvan,,,,
13/10/2018,Ged,Razvan,,,
20/10/2018,Razvan,,Ged,Razvan,
27/10/2018,Ged,Ged,,,
03/11/2018,Razvan,,,,
10/11/2018,Ged,Razvan,,,
17/11/2018,Razvan,,Razvan,Ged,
24/11/2018,Ged,Ged,,,
01/12/2018,Razvan,,,,
08/12/2018,Ged,Razvan,,,
15/12/2018,Razvan,,Ged,Razvan,
